src/ProofOfConcepts/ACLFilterChips/customStyles.css

```diff
@@ -1,38 +1,23 @@
-.pf-c-label.mas-m-solid.smaller-label {
+.pf-c-label.mas-m-solid.mas-m-text {
```

Comment:
I'm not sure this extra `mas-m-text` class name is necessary. The styles defined here are what I would expect as part of a solid modifier class (e.g. transparent border and white text). This might be something to look into and see if `mas-m-text` is actually needed or not.

Reply:
It looks like the transparent border and white text are needed, but I took out everything else in it as it seems okay to.

Reply:
I agree that the styles defined for this rule are needed. I'm not sure that we need both classes `mas-m-solid` and `mas-m-text`. If you remove `mas-m-text` from this, will the styles still be applied as expected? The class name `mas-m-solid` describes the styles that we're defining here (i.e. to create a label with a solid fill, we need to make the border transparent and the text white).

Reply:
when mas-m-text is removed, the text and border don't appear as we would like: 
**just "mas-m-text" removed**
![Screenshot from 2022-08-31 14-24-41](https://user-images.githubusercontent.com/72694484/187689419-645e49f9-2501-4305-b379-8c1c139849cb.png)

and when the mas-m-solid is removed the text and border are also affected, and the background color of the button too:
**just "mas-m-solid" removed**
![Screenshot from 2022-08-31 14-30-44](https://user-images.githubusercontent.com/72694484/187690387-000ad1fd-c74d-4b84-a9de-96e96149e3bc.png)

Reply:
I think the issue is with the specificity, where the styles defined for a PF rule like `.pf-c-label.pf-m-blue` are being applied over what we're trying to define for `.pf-c-label.mas-m-solid`. 

That's not what I would expect, I would expect that whatever custom classes we define would override something with equal specificity in the PF styles. 

@dlabaj - Do you have any suggestions on how to add a custom class that doesn't get overridden by PF styles of equal specificity? Is this more related to how we're pulling in styles?

Reply:
It has to do with the import order.  You need to  import your css in your js files last.  Then it works correctly.